### PR TITLE
Explain how to install plugins for people unfamiliar with Neo4j

### PR DESCRIPTION
**Pull request by catherineluse**
# Description

This docs change would help me install the required plugin to be able to use DateTimes.

# Issue

[Please provide a link to the GitHub issue in which the proposal for this work was discussed below. ](https://github.com/neo4j/graphql/issues/941)

It's important that changes are discussed in advance to make sure we choose the right solution and to avoid duplication of effort.

Closes:

- 

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] TCK tests have been updated
- [ ] Integration tests have been updated
- [ ] Example applications have been updated
- [ ] New files have copyright header
- [ ] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by darrellwarde** [comment]
Hey @catherineluse, thanks for the contribution, but I'm not such a fan of pointing this out as it's very specific to Neo4j Desktop. If we need to link to this, it would be preferably to a page like https://neo4j.com/labs/apoc/4.4/installation/ which covers all bases.

**Comment by catherineluse** [comment]
I have updated the PR and signed the CLA

File: docs/modules/ROOT/pages/introduction.adoc [hunk 1]
```diff
@@ -68,6 +68,7 @@ Additionally, prerelease version numbers may have additional suffixes, for examp
   - *4.2:* 4.2.9+ with APOC 4.2.0.8+
   - *4.3:* 4.3.2+ with APOC 4.3.0.3+
   - *4.4:* 4.4.0+ with APOC 4.4.0.1+
+  For help installing the APOC plugin, see https://neo4j.com/labs/apoc/4.4/installation/[this page.]
 2. https://nodejs.org/en/[Node.js] 12+
```
Comment (darrellwarde) [inline docs/modules/ROOT/pages/introduction.adoc:73]:
```suggestion
2. https://nodejs.org/en/[Node.js] 12+

For help installing the APOC plugin, see https://neo4j.com/labs/apoc/4.4/installation/[this page.]

```

**Review by darrellwarde — CHANGES_REQUESTED** [review]
This will render quite funny as it is, so will just move this down a couple of lines.

**Review by darrellwarde — APPROVED** [review]

**Pull request merged**